R/inverse_interpolation.R

```diff
@@ -1,6 +1,6 @@
 #' Local inverse interpolation
 #'
-#' @param x The matrix of "training" values of x, with one row per training set.
+#' @param x The matrix of "training" values of x, with one item per training set.
```

Comment:
I think this should be `row`, even though if `x` is one-dimensional, passing a vector is OK.

Reply:
OK, reverted.